api/test.php

```diff
@@ -0,0 +1,179 @@
+		$this->siteUrl = trim($siteUrl, " /\t\n\r\0\x0B");
+		$this->apiKey = $apiKey;
+	}
+
+	/**
+	 * Convert given error info (error message + error HTTP code) into a JSON object
+	 * and return it as a JSON-encoded string.
+	 */
+	static private function errorToJson($message, $code) {
+		return json_encode(array(
+			'error' => $message,
+			'code' => $code
+		));
+	}
+
+	/**
+	 * Make an API call.
+	 * @param string $route - API entry.
+	 * @param array $params - associative array containing values for API entry parameters.
+	 * @return string - JSON-encoded response.
+	 */
+	private function call($route, $params) {
+		// References (to use CURL):
+		// https://stackoverflow.com/a/48896992
+		// https://codular.com/curl-with-php
+		if (!isset(ApiCall::$routes[$route]))
+			return $this->errorToJson('Unknown route.', 500);
+		$method = ApiCall::$routes[$route];
+		$authorization = "Authorization: Bearer ".$this->apiKey;
+		$headers = array($authorization);
+		$curl = curl_init();
+		$url = $this->siteUrl.'/api.php?route='.$route;
+		if ($method == 'JSON') {
+			$json_string = json_encode($params);
+			curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "POST");
+			curl_setopt($curl, CURLOPT_POSTFIELDS, $json_string);
+			$headers[] = 'Content-Type: application/json';
+		} else if ($method == 'POST') {
+			curl_setopt($curl, CURLOPT_POST, 1);
+			curl_setopt($curl, CURLOPT_POSTFIELDS, $params);
+		} else if (!empty($params)) {
+			$paramsArray = array();
+			foreach ($params as $key => $value)
+				$paramsArray[] = urlencode($key).'='.urlencode($value);
+			$url .= '&'.implode('&', $paramsArray);
+		}
+		curl_setopt($curl, CURLOPT_HTTPHEADER, $headers);
+		curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+		curl_setopt($curl, CURLOPT_URL, $url);
+		$response = curl_exec($curl);
+		$httpCode = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+		if ($httpCode != 200)
+			$response = $this->errorToJson($response, $httpCode);
+		curl_close($curl);
+		return $response;
+	}
+
+	/**
+	 * Make an API call to entry `players/cd`.
+	 */
+	public function listGamesWithPlayersInCD() {
+		return $this->call('players/cd', array());
+	}
+
+	/**
+	 * Make an API call to entry `players/missing_orders`.
+	 */
+	public function listGamesWithMissingOrders() {
+		return $this->call('players/missing_orders', array());
+	}
+
+	/**
+	 * Make an API call to entry `game/status`.
+	 */
+	public function getGamesStates($gameID, $countryID) {
+		return $this->call('game/status', array('gameID' => $gameID, 'countryID' => $countryID));
+	}
+
+	/**
+	 * Make an API call to entry `game/orders`.
+	 */
+	public function setOrders($gameID, $turn, $phase, $countryID, $orders, $ready = null) {
+		return $this->call(
+			'game/orders',
+			array(
+				'gameID' => $gameID,
+				'turn' => $turn,
+				'phase' => $phase,
+				'countryID' => $countryID,
+				'orders' => $orders,
+				'ready' => $ready
+			));
+	}
+}
+
+/**
+ * Usage example of ApiCall.
+ * Make call to all current API entries.
+*/
```

Comment:
This definitely needs to be disabled and only active if a Config:: value is set please